.github/workflows/image.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
         - name: Generate Version
           id: version
           run: |
-            version="$(cat VERSION)-$(date +%y%m%d%H%M%S)"
+            version="$(cat VERSION)-$(date +%Y-%m-%d-%H:%M:%S)"
```

Comment:
Have you tried making images using this and also tried bundle run? 
Could you please point to a CI run? 

I think all non-trivial CI (especially those that are run after merge) changes must be tested in a fork

Reply:
Absolutely I agree🙂. It is best that we highlight this in the Developer Docs section.